numba/generators.py

```diff
@@ -211,24 +217,17 @@ def lower_finalize_func_body(self, builder, genptr):
         Lower the body of the generator's finalizer: decref all live
         state variables.
```

Comment:
Is this docstring still accurate about decref'ing the state variables now that the second block of code has been removed?